docs/concepts/writing-config.md

```diff
@@ -0,0 +1,433 @@
+---
+title: Writing Istio Configuration
```

Comment:
Make the title shorter so it tends to fix in a tab. The tab will always have Istio / in it, so no need to repeat Istio again.